### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by user18130814200115-2**
Added sections on installing, building, using and troubleshooting
In essence, I described the process I went through to get wlpinyin working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,43 @@ No gui. Candidate are showed in text area. Cuz i really can not find a working n
...
+```
+sess: 0, msgtype: deploy, msg: start
+sess: 0, msgtype: deploy, msg: failure
+```
+You are probably missing rime config files, see step 1 of Dependencies.
+If characters show up as boxes, also check your rime config.
+If you get an error saying that wlpinyin cannot find a file, check your rime config.
+
+If wlpinyin works for you in most cases but not with certain programs, then you might notify the application developer.
+Applications such as Chromium are notorious for now working with many other input methods such as fcitx.
```
Comment (xhebox) [inline README.md:49]: `now` -> `not`?

File: README.md [hunk 2]
```diff
@@ -7,3 +7,43 @@ No gui. Candidate are showed in text area. Cuz i really can not find a working n
 Now everything is hardcoded, but i except it can be configured like dwm. It is overkill to add IPC or dynamic config in such a project, though, i am writing it with modularity in my mind.
 
 It will not be too hard to add new input backend engine, enable it by pkgconf check and ifdef at compilation time.
+
+## Install
+### Dependencies
+
+1. Make use you have [rime](https://github.com/rime/librime) installed
+2. Place your desired rime configuration files in ~/.config/wlpinyin/
+    - [Rime's minimal example](https://github.com/rime/librime/tree/master/data/minimal) is a good place to start.
```
Comment (xhebox) [inline README.md:16]: I want to mention that [brise](https://github.com/rime/brise) may be needed for bootstrap. You don't really need to config manually the first time starting.

I am not sure. I remembered that I don't need to config manually the last time of testing.

File: README.md [hunk 3]
```diff
@@ -7,3 +7,43 @@ No gui. Candidate are showed in text area. Cuz i really can not find a working n
...
+If you get an error along the lines of
+```
+sess: 0, msgtype: deploy, msg: start
+sess: 0, msgtype: deploy, msg: failure
+```
+You are probably missing rime config files, see step 1 of Dependencies.
+If characters show up as boxes, also check your rime config.
+If you get an error saying that wlpinyin cannot find a file, check your rime config.
+
+If wlpinyin works for you in most cases but not with certain programs, then you might notify the application developer.
```
Comment (xhebox) [inline README.md:48]: Specifically, it is `text-input-v3` protocol for applications and `input-method-v2` for compositors. With these protocols supported, wlpinyin can be used.

**Review by xhebox — COMMENTED** [review]
overall LGTM, excellent work!

File: README.md (hunk 3 above)
Comment (user18130814200115-2) [reply README.md:48]: That is good information, I had a vague idea of this requirement, but I was not sure of the exact protocols required

Comment (user18130814200115-2) [reply README.md:48]: added to patch 1

File: README.md (hunk 1 above)
Comment (user18130814200115-2) [reply README.md:49]: added to patch 1

File: README.md (hunk 2 above)
Comment (user18130814200115-2) [reply README.md:16]: I am personally unfamiliar with brise, it looks like it is a system to automate the configuration of rime, but it looks depricated.
In other words, it seems like you do not stricly need brise, I will add a section to patch 1, you say if you are happy with it or not :)

Comment (xhebox) [reply README.md:16]: Yes, you don't strictly need brise. It is just common files that will copied into your local config directory.

Comment (xhebox) [reply README.md:16]: I think it is fine not to mention that. If you can compile it, you should be able to config rime by yourself.

**Comment by xhebox** [comment]
Fix your typo `brice` -> `brise`. Merged.

**Pull request merged**